rest_service.go

```diff
@@ -35,10 +48,24 @@ func NewManagerService(m Manager) *Service {
 	}
 }
 
+const (
+	// DefaultCachePruneDelay is the duration between LRU cache prunes.
+	DefaultCachePruneDelay = 10 * time.Second
+	// DefaultMaxCacheSize is the maximum allowed size of the LRU cache.
+	DefaultMaxCacheSize = 1024 * 1024 * 1024
+)
+
 // App constructs and returns a gimlet application for this
 // service. It attaches no middleware and does not start the service.
 func (s *Service) App() *gimlet.APIApp {
 	s.hostID, _ = os.Hostname()
+	s.cache = lru.NewCache()
+	s.cacheMutex.Lock()
```

Comment:
If you don't need to unlock a mutex before the function returns, the Go idiom is:
```
mu.Lock()
defer mu.Unlock()
```
It makes it a bit easier to not forget to unlock the mutex.

Reply:
Changed to defer.